specification/network/resource-manager/Microsoft.Network/stable/2018-02-01/expressRouteCrossConnection.json

```diff
@@ -0,0 +1,738 @@
+            "description": "The name of the ExpressRouteCrossConnection."
+          },
+          {
+            "name": "peeringName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the peering."
+          },
+          {
+            "name": "devicePath",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The path of the device."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+              "description": "Request successful. The operation returns the resulting ExpressRouteCrossConnectionsRouteTable resource.",
+              "schema": {
+                "$ref": "./expressRouteCircuit.json#/definitions/ExpressRouteCircuitsRoutesTableListResult"
+              }
+          },
+          "202": {
+            "description": "Accepted and the operation will complete asynchronously."
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    }
+  },
+  "definitions": {
+    "ExpressRouteCircuitReference": {
+      "properties": {
+        "id": {
+          "type":"string",
+          "description": "Corresponding Express Route Circuit Id."
+        }
+      }
+    },
+    "ExpressRouteCrossConnectionServiceProviderProperties": {
+      "properties": {
+        "serviceProviderName": {
+          "type": "string",
+          "description": "The serviceProviderName."
+        },
+        "peeringLocation": {
+          "type": "string",
+          "description": "The peering location of the ExpressRoute circuit."
+        },
+        "bandwidthInMbps": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The circuit bandwidth In Mbps."
+        }
+      },
+      "description": "Contains ServiceProviderProperties in an ExpressRouteCrossConnection"
+    },
+    "ExpressRouteCrossConnectionPropertiesFormat": {
+      "properties": {
+        "primaryAzurePort": {
+          "type": "string",
+          "description": "The name of the primary  port."
+        },
+        "secondaryAzurePort": {
+          "type": "string",
+          "description": "The name of the secondary  port."
+        },
+        "sTag": {
+          "type": "integer",
+          "description": "The identifier of the circuit traffic."
+        },
+        "serviceProviderProperties": {
+          "$ref": "#/definitions/ExpressRouteCrossConnectionServiceProviderProperties",
+          "description": "The ServiceProviderProperties."
+        },
+        "expressRouteCircuit": {
+          "$ref": "#/definitions/ExpressRouteCircuitReference",
+          "description": "The ExpressRouteCircuit"
+        },
+        "serviceProviderProvisioningState": {
+          "type": "string",
```

Comment:
Provisioning states are usually readonly. Please confirm.

Reply:
The serviceProviderProvisioning state is writeable by the ExpressRoute Provider subscription.